### PR TITLE
Remove package-lock.json and yarn.lock

### PR DESCRIPTION
**Pull request by lethemanh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,22 +3,20 @@
...
-    "react-bootstrap": "^1.5.2",
-    "react-dom": "^17.0.2",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0",
+    "react-dom": "17.0.2",
+    "react-router": "5.2.0",
+    "react-router-dom": "5.2.0",
     "react-router-guards": "1.0.0",
     "react-scripts": "4.0.3",
     "web-vitals": "^1.0.1"
```
Comment (ppppbn) [inline package.json]: you forgot this

File: package.json [hunk 2]
```diff
@@ -3,22 +3,20 @@
...
+    "@testing-library/jest-dom": "5.14.1",
+    "@testing-library/react": "11.2.7",
+    "@testing-library/user-event": "12.8.3",
+    "antd": "4.16.7",
+    "axios": "0.21.1",
+    "dotenv-safe": "8.2.0",
+    "moment": "2.29.1",
+    "node-sass": "6.0.1",
+    "prop-types": "15.7.2",
     "react": "^17.0.2",
```
Comment (ppppbn) [inline package.json]: and here

**Review by nddmanh — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -3,25 +3,23 @@
...
+    "@testing-library/user-event": "12.8.3",
+    "antd": "4.16.7",
+    "axios": "0.21.1",
+    "dotenv-safe": "8.2.0",
+    "moment": "2.29.1",
+    "node-sass": "6.0.1",
+    "prop-types": "15.7.2",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-router": "5.2.0",
```
Comment (ppppbn) [inline package.json]: Also, do we need 'react-router'?

**Comment by ppppbn** [comment]
You have two commits with the same message.

**Comment by lethemanh** [comment]
> You have two commits with the same message.

**Review by ppppbn — APPROVED** [review]

**Pull request merged**